src/components/CurriculumComponents/CurricVisualiserFilters/CurricFiltersTiers.tsx

```diff
@@ -43,9 +43,10 @@ export function CurricFiltersTiers({
         <OakBox>
           <OakHeading
             id={"tiers-label"}
-            tag="h4"
-            $font={"heading-6"}
-            $mb="space-between-s"
+            tag="h3"
```

Comment:
Note that these will be changing the desktop styling also, does this still look correct in desktop?